.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ default_language_version:
     python: python3
 repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.6.0
+    rev: v5.0.0
```

Comment:
@chandanchowdhury does this cause anything to break on dev.Dockerfile? Asking since this is a major version change and the dev.Dockerfile uses Python 3.10.